.github/ISSUE_TEMPLATE/90_bug_report.yml

```diff
@@ -0,0 +1,104 @@
+name: Bug report
+description: File a bug report. If you need help, please join our [Discord server](https://discord.com/invite/ollama).
+labels: [needs-triage, bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please check if your bug is [already filed](https://github.com/ollama/ollama/issues) before filing a new one.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What is the issue?
+      description: What happened? What did you expect to happen?
+    validations:
+      required: true
+  - type: textarea
+    id: what-was-expected
+    attributes:
+      label: What did you expect to see?
+      description: What did you expect to see/happen instead?
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to reproduce
+      description: What are the steps you took that hit this issue?
+    validations:
+      required: false
+  - type: textarea
+    id: changes
+    attributes:
+      label: Are there any recent changes that introduced the issue?
+      description: If so, what are those changes?
+    validations:
+      required: false
+  - type: dropdown
+    id: os
+    attributes:
+      label: OS
+      description: What OS are you using? You may select more than one.
+      multiple: true
+      options:
+        - Linux
+        - macOS
+        - Windows (Native)
+        - Windows (WSL)
+        - Other
+    validations:
+      required: false
```

Comment:
This is kind of a duplicate of `os-version` below. What's missing is actually platform: os (macOS, Windows, Linux), architecture (amd64, arm64), and virtualization (Docker, WSL2, VM)

Reply:
+1 on adding Docker and WSL they have their own unique issues

Reply:
I split them out.